recipe/meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "build": "pyh28a5fc4_0",
+  "build": "pyh233a6f4_0",
```

Comment:
Is this a version of the build? I'm not sure exactly what this is for and after doing some basic google searching for `pyh233a6f4_0` I wasn't able to find anything online. Is this something specific to our repo? Any context would help here

Reply:
I'm interested in this, too! How does meta.json get populated in general? Is it a manual process?

Reply:
https://github.com/ufs-community/workflow-tools/blob/develop/recipe/README.md